instrumentation/afl-compiler-rt.o.c

```diff
@@ -2761,5 +2761,11 @@ void __afl_injection_xss(u8 *buf) {
 
 }
 
+void __afl_ijon_set(u32 edge) {
```

Comment:
This is conceptually very close to `__afl_coverage_interesting(u8 val, u32 id)`, however that one doesn't use modulo. I think the functions should behave consistently here.
Also, should it be named `ijon`, or is there a more descriptive name?
 `https://github.com/AFLplusplus/AFLplusplus/blob/2a489f844b7bc910caad2ab46aa3d86c3e6fbef8/instrumentation/afl-compiler-rt.o.c#L2705

Reply:
I agree, IMHO the modulo should be in the frida code, and yes, this could use the __afl_coverage_interesting instead.

Reply:
Indeed they are similar :thinking:, but the usage is different. Do you use `__afl_coverage_interesting` to mark a certain code block/instrucion as particularly "interesting" by setting an high value (ex. 128)? 

It could be named `__afl_coverage_set` (in practice) or `__afl_state_set` (conceptually).
Ideally the edge/id is something that can be set by the user (that's why the modulo), however I'm not really sure what are the implications when using source code instrumentation. I will try to do some tests